packages/plugin-core/assets/dendron-ws/tutorial/tutorial.md

```diff
@@ -0,0 +1,34 @@
+---
+id: 01u0co3RYjOM1bjpc2qIU
+title: Tutorial
+desc: ''
+updated: 1624452370349
+created: 1624333847315
+---
+
+## Welcome to Dendron
+
+This guide will help you learn the basics of Dendron. You can also follow the Getting Started guides on our [website](https://wiki.dendron.so/notes/e86ac3ab-dbe1-47a1-bcd7-9df0d0490b40.html).
+
+If you close VS Code, you can always get back to this workspace by going to `File > Open Workspace` in VS Code's menu and then selecting your workspace file: [[todo]].
+
+The tutorial steps take about 10 minutes to complete. Let's get started!
```

Comment:
we talked about this in the morning, would be great if we can get this under 5min